Nodejs/Product/Nodejs/Options/NodejsNpmOptionsPage.cs

```diff
@@ -47,6 +48,8 @@ public NodejsNpmOptionsPage()
         /// </summary>
         public bool ShowOutputWindowWhenExecutingNpm { get; set; }
 
+        public bool EnableAutomaticTypingsAcquisition { get; set; }
```

Comment:
Hmm... I may have mislead you here... I just checked on Dev15 and turns out that they don't expose any option in the UI - currently they only plan to expose it through a typingOptions section in a project-specific jsconfig.json (which we won't support in Dev14). So for now, I think that means we have a couple options to reduce confusion:
* only display the option in Dev14
* remove the option altogether

I'm leaning towards the latter, but it's your call.